File: tests/Generator/resource/python/default/author.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler, Tag
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, Annotated, Union
from .location import Location


# An simple author element with some description
class Author(BaseModel):
    title: str = Field(alias="title")
    email: Optional[str] = Field(default=None, alias="email")
    categories: Optional[List[str]] = Field(default=None, alias="categories")
    locations: Optional[List[Location]] = Field(default=None, alias="locations")
    origin: Optional[Location] = Field(default=None, alias="origin")
    pass

File: tests/Generator/resource/csharp/default/Author.cs
using System.Text.Json.Serialization;

/// <summary>
/// An simple author element with some description
/// </summary>
public class Author
{
    [JsonPropertyName("title")]
    public string Title { get; set; }

    [JsonPropertyName("email")]
    public string? Email { get; set; }

    [JsonPropertyName("categories")]
    public System.Collections.Generic.List<string>? Categories { get; set; }

    [JsonPropertyName("locations")]
    public System.Collections.Generic.List<Location>? Locations { get; set; }

    [JsonPropertyName("origin")]
    public Location? Origin { get; set; }

}

